pkg/cli/admin/ocpcertificates/regeneratemco/updatesecrets.go

```diff
@@ -47,8 +47,9 @@ func (o *RegenerateMCOOptions) updateSecrets(ctx context.Context) error {
 		return fmt.Errorf("cannot list MAO secrets: %w", err)
 	}
 	for _, secret := range secretList.Items {
-		// These two are managed by the MCO but unused. Skip them since the MCO will write them back.
-		if secret.Name == mcoManagedWorkerSecret || secret.Name == mcoManagedMasterSecret {
+		// Managed secrets should be created by the MCO but are unused. Normally they would be written back,
+		// but baremetal IPI uses the managed worker secret. Skip only the Master secret for now.
```

Comment:
But won't the MCO just rewrite the `worker-user-data-managed` secret back next time `syncRenderConfig()` runs?  That's kind of the idea, it's managed state...so to fix this properly we need to sync the `-managed` logic in the MCO right?

Reply:
Yes, I was wondering if BM did something different, but otherwise I am updating this locally.

Essentially to do this via oc we would need to update the root-ca configmap which unfortunately will trigger another reboot. We can also patch this via the MCO but that would require another payload to update to which I am not sure is feasible for older versions